pkg/handler/send.go

```diff
@@ -70,6 +70,8 @@ var _ = RequestSchema.Add("TemplateVariables", sms_infra.TemplateVariablesSchema
 type Result struct {
 	ClientResponse string `json:"client_response"`
 	Success        bool   `json:"success"`
+	HasNumSegments bool   `json:"has_num_segments"`
+	NumSegments    int    `json:"num_segments"`
```

Comment:
Change to 

```
SegmentCount *int `json:"segment_count,omitempty"`
```

So that if you do not set it, then it is nil, and `omitempty` will make it absent in the JSON document.